alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html

```diff
@@ -1,9 +1,23 @@
-<div mat-dialog-title>
-  <h2 class="card-title">{{ currentCard.fileNumber }} ({{ currentCard.applicant }})</h2>
-  <span>
-    in <span class="italic card-status-label">{{ currentCard.statusDetails.label }}</span>
-  </span>
-  <span class="pull-right card-state">{{ currentCard.paused ? 'Paused' : 'Active' }}</span>
+<div mat-dialog-title class="header">
+  <div>
+    <h2 class="card-title">{{ currentCard.fileNumber }} ({{ currentCard.applicant }})</h2>
+    <span>
+      in <span class="italic card-status-label">{{ currentCard.statusDetails.label }}</span>
+    </span>
+  </div>
+  <div class="center">
+    <div class="card-state">{{ currentCard.paused ? 'Paused' : 'Active' }}</div>
+    <div>
+      <button (click)="onTogglePriority()" mat-icon-button>
+        <span
+          [ngClass]="{
+            'high-priority': currentCard.highPriority,
+            'low-priority': !currentCard.highPriority
```

Comment:
should this be just regular priority?

Reply:
Is there a chance of getting more than 2 priorities? If yes maybe a priority code will work better here

Reply:
Renamed the css class to empty-priority and filled-priority to better match what they mean